File: Lab3/MyFirstApplication/app/src/main/java/com/example/myfirstapplication/models/ChatUser.kt
package com.example.myfirstapplication.models

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "chat_users")
data class ChatUser(

    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,

    val chatId: Int,
    val userId: Int
)

File: Lab4/LW4/LW4/Data/Models/ChatUser.cs
namespace LW4.Data.Models;

public class ChatUser
{
    public int Id { get; set; }
    public int ChatId { get; set; }
    public virtual Chat Chat { get; set; } = null!;

    public int UserId { get; set; }
    public virtual User User { get; set; } = null!;
}
